flink-table/flink-table-common/src/main/java/org/apache/flink/table/connector/source/abilities/SupportsDynamicFiltering.java

```diff
@@ -60,9 +60,16 @@
 public interface SupportsDynamicFiltering {
 
     /**
-     * Applies the candidate filter fields into the table source, and return the accepted fields.
-     * The data corresponding the filter fields will be provided in runtime, which can be used to
-     * filter the partitions or the input data.
+     * Return the filter fields this partition table source supported. This method is used in
+     * planner, which can be used to judge whether the source table is a suitable partition table
+     * for filtering partitions.
      */
-    List<String> applyDynamicFiltering(List<String> candidateFilterFields);
+    List<String> listAcceptedFilterFields();
+
+    /**
+     * Applies the candidate filter fields into the table source. The data corresponding the filter
+     * fields will be provided in runtime, which can be used to filter the partitions or the input
+     * data.
+     */
```

Comment:
/**
     * Applies the candidate filter fields into the table source. The data corresponding the filter
     * fields will be provided in runtime, which can be used to filter the partitions or the input
     * data.
     *
     * `<p>`NOTE: the candidate filter fields are always from the result of {@link
     * #listAcceptedFilterFields()}.
     */

Reply:
> /** * Applies the candidate filter fields into the table source. The data corresponding the filter * fields will be provided in runtime, which can be used to filter the partitions or the input * data. * * `<p>`NOTE: the candidate filter fields are always from the result of {@link * #listAcceptedFilterFields()}. */

Done!